docs/DeveloperGuide.md

```diff
@@ -1,34 +1,845 @@
-# Developer Guide
+By: `Team M16-1` Since: `Jan 2020` License: `MIT`
 
-## Design & Implementation
```

Comment:
The format of the page looks weird feels like there is some space wasted on both sides making it difficult to read. Maybe it would be good to change to a different theme?